app/konnect/runtime-manager/kong-gateway-runtime.md

```diff
@@ -206,6 +207,88 @@ Manager overview.
 
     The Runtime Manager will include a new entry for your instance.
 
+{% endnavtab %}
+{% navtab Helm %}
+
+1. If you already have a {{site.base_gateway}} instance configured with Helm,
+move onto the next step.
+
+    If this is a new instance, set up the following:
+
+    1. Create a namespace:
+        ```sh
+        $ kubectl create namespace kong
+        ```
+
+    2. Add the Kong charts repository:
+        ```bash
+        $ helm repo add kong https://charts.konghq.com
+        ```
+
+    3. Update Helm:
+        ```bash
+        $ helm repo update
+        ```
+
+2. Create a `tls` secret using the files you saved earlier:
+
+    ```bash
+    $ kubectl create secret tls kong-cluster-cert \
+      --cert=/tmp/cluster.crt \
+      --key=/tmp/cluster.key
```

Comment:
The dedicated `tls` Secret type only supports cert/key pairs, so CA certificates where you only have the certificate need to go into a generic certificate instead:

```suggestion
    $ kubectl create secret tls kong-cluster-cert \
      --cert=/tmp/cluster.crt \
      --key=/tmp/cluster.key

    $ kubectl create secret generic kong-cluster-ca \
      --from-file=ca.crt=/tmp/ca.crt
```
Technically speaking they could go into a ConfigMap instead, but our mount mechanism doesn't support those yet.